lib/UserInterface/textinput/TextInput.cpp

```diff
@@ -61,9 +62,11 @@ namespace textinput {
     }
     fContext->GetEditor()->ResetText();
 
-    // Signal displays that the input got taken.
-    std::for_each(fContext->GetDisplays().begin(), fContext->GetDisplays().end(),
-             std::mem_fun(&Display::NotifyResetInput));
+    if (!fIsDumbTerm) {
+        // Signal displays that the input got taken.
+        std::for_each(fContext->GetDisplays().begin(), fContext->GetDisplays().end(),
+                      std::mem_fun(&Display::NotifyResetInput));
```

Comment:
The `if` should test the display of the current iteration: if it's dumb, don't call NotifyResetInput..